src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/factory/BedMoveEntityFactory.kt

```diff
@@ -0,0 +1,55 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.factory
+
+import io.github.bluegroundltd.kfactory.Factory
+import io.github.bluegroundltd.kfactory.Yielded
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.BedEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.BedMoveEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity.BookingEntity
+import uk.gov.justice.digital.hmpps.approvedpremisesapi.util.randomDateTimeBefore
+import java.time.OffsetDateTime
+import java.util.UUID
+// WF: need some assistance with Detekt file
```

Comment:
we encourage use of @Suppress if detekt is complaining about something you disagree with (e.g. add an additional entry to the annotation below this comment.

The detekt file should have been used to set a baseline when detekt was first introduced (to avoid lots of rework), but people continued to update it. At some point it would be nice to drop it (or at least regenerate it, because it'll have lots of out of date references in it that are no longer applicable)

Reply:
I think i can drop this todo. will update in the next pass.

Reply:
oh wait - sure, may want to talk to you about this one, to get that right.

Reply:
comment removed and suppression is in place as discussed. we will not add to the detekt file.